src/pip/_internal/commands/__init__.py

```diff
@@ -108,6 +101,7 @@ def create_command(name, **kwargs):
 
 
 def get_similar_commands(name):
+    # type: (str) -> Union[bool, str]
```

Comment:
`False` here is only to express “no guess available” and all usages of this function does something like

```python
guess = get_similar_commands(...)
if guess:
   ...
```

So it’d be more pragmatic to change this to return `None` or `""` when no matches are available instead.

Reply:
Yeah, that is better. Handled in #9410.